From: SeongJae Park <sj@kernel.org>
Date: Tue, 15 Apr 2025 19:55:35 -0700
Subject: [PATCH] tools/mm: add thp_swap_allocator_test to .gitignore

Commit 95139d940845 ("tools/mm: introduce a tool to assess swap entry
allocation for thp_swapout") didn't add the executable output file to
.gitignore file.  Ignore it to make 'git status' output clean.

Fixes: 95139d940845 ("tools/mm: introduce a tool to assess swap entry allocation for thp_swapout")
Signed-off-by: SeongJae Park <sj@kernel.org>
---
 tools/mm/.gitignore | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/mm/.gitignore b/tools/mm/.gitignore
index 922879f93fc8..1446a659e540 100644
--- a/tools/mm/.gitignore
+++ b/tools/mm/.gitignore
@@ -2,3 +2,4 @@
 slabinfo
 page-types
 page_owner_sort
+thp_swap_allocator_test
-- 
2.39.5

